thread/src/components/common/Input.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import { TextInput, View, Text } from "react-native";
+
+const Input = ({ label, value, onChangeText, secureTextEntry }) =>
+{
+
+    const { inputStyle, labelStyle, containerStyle } = styles;
+
+    return (
+
+        <View style={ containerStyle }>
+            <Text style={ labelStyle }>{ label }</Text>
```

Comment:
This has to empty, you use it like this.

```
<Input
    onChangeText={onChangeText}
    secureTextEntry={secure}
/>
```

Where is the label? Maybe I need glasses after all